h/api/test/schemas_test.py

```diff
@@ -237,18 +237,8 @@ def test_it_raises_if_group_is_not_a_string(self):
 
         assert str(err.value) == "group: False is not of type 'string'"
 
-    def test_it_raises_if_permissions_is_missing(self):
```

Comment:
Sorry, this a big diff and I can't find it. What's the context here? Is permissions no longer required? If so, how so?